ledger-service/http-json/BUILD.bazel

```diff
@@ -577,69 +734,60 @@ test_suite(
 ]
 
 [
-    da_scala_test_suite(
-        name = "failure-tests-{}".format(edition),
+    da_scala_library(
+        name = "failure-tests-pkg-{}".format(edition),
         srcs = glob(["src/failure/scala/**/*.scala"]),
-        data = [
-            ":Account.dar",
-            "//docs:quickstart-model.dar",
-            "//ledger/test-common:dar-files",
-            "//ledger/test-common/test-certificates",
-            "@toxiproxy_dev_env//:bin/toxiproxy-cmd" if not is_windows else "@toxiproxy_dev_env//:toxiproxy-server-windows-amd64.exe",
-        ],
-        # See https://github.com/digital-asset/daml/issues/9886
-        flaky = True,
         plugins = [
             kind_projector_plugin,
         ],
-        resources = glob(["src/failure/resources/**/*"]),
         scala_deps = [
             "@maven//:com_typesafe_akka_akka_http_core",
-            "@maven//:com_typesafe_scala_logging_scala_logging",
+            #"@maven//:com_typesafe_scala_logging_scala_logging",
             "@maven//:io_spray_spray_json",
+            "@maven//:org_scalactic_scalactic",
             "@maven//:org_scalatest_scalatest_core",
             "@maven//:org_scalatest_scalatest_freespec",
             "@maven//:org_scalatest_scalatest_matchers_core",
             "@maven//:org_scalatest_scalatest_shouldmatchers",
             "@maven//:org_scalaz_scalaz_core",
-            "@maven//:org_typelevel_cats_core",
-            "@maven//:org_typelevel_cats_effect",
+            #"@maven//:org_typelevel_cats_core",
+            #"@maven//:org_typelevel_cats_effect",
```

Comment:
Please remove instead of commenting out.

Reply:
Done.